src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -470,6 +492,33 @@ public String getQuestionResultStatisticsCsv(
                + ", Average Points" + Const.EOL
                + fragments + Const.EOL;
     }
+    
+    /**
+     * Used as helper method to put recipient entries from an unsorted map to a sorted map
```

Comment:
Follow the header comment convention for methods. `Puts ...`. Also, don't explain who is using it for what; simply explain what it does. Is this method doing any sorting? If not, the method name should not contain `sort`